### PR TITLE
MEN-4097: Document mender-shell integration variables

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: 04.System-updates-Debian-family/99.Variables/docs.md [hunk 1]
```diff
@@ -111,10 +111,10 @@ The size of the boot partition.
 
 #### `MENDER_CLIENT_VERSION`
 
-<!--AUTOVERSION: "Value: %"/ignore-->
-> Value: master (default)
+> Value: latest (default)
 
-The version of the Mender client to include in the update.
+<!--AUTOVERSION: ", \"%\" or a specific"/ignore-->
+The version of the Mender client to include in the update. Valid values are "latest" (default), "master" or a specific version.
```
Comment (merlin-northern) [inline 04.System-updates-Debian-family/99.Variables/docs.md:117]: maybe a 
* list 
* with bullets ?

Comment (lluiscampos) [reply 04.System-updates-Debian-family/99.Variables/docs.md:117]: By looking at the other variables in this file, to be consistent we should instead have `Value: latest(default)/master/<version>`.

I'll rework accordingly.

Comment (lluiscampos) [reply 04.System-updates-Debian-family/99.Variables/docs.md:117]: Done.

**Comment by lluiscampos** [comment]
See https://github.com/mendersoftware/mender-convert/pull/263

File: 04.System-updates-Debian-family/99.Variables/docs.md [hunk 2]
```diff
@@ -117,6 +117,21 @@ The size of the boot partition.
 The version of the Mender client to include in the update.
 
 
+#### MENDER_ADDON_SHELL_INSTALL
+
+> Value: y/n(default)
+
+Install Mender Shell addon.
```
Comment (merlin-northern) [inline 04.System-updates-Debian-family/99.Variables/docs.md:124]: suggested change: "Install Mender Shell addon." -> "Install the Mender Shell addon."

File: 04.System-updates-Debian-family/99.Variables/docs.md [hunk 3]
```diff
@@ -117,6 +117,21 @@ The size of the boot partition.
...
+
+
+#### MENDER_ADDON_SHELL_VERSION
+
+<!--AUTOVERSION: "/%/"/ignore-->
+> Value: latest(default)/master/&lt;version&gt;
+
+The version of the Mender Shell addon to include in the update.
```
Comment (merlin-northern) [inline 04.System-updates-Debian-family/99.Variables/docs.md:132]: suggested change: "addon" -> "add-on"

**Review by merlin-northern — APPROVED** [review]
two suggested changes. lgtm otherwise.

File: 04.System-updates-Debian-family/99.Variables/docs.md (hunk 3 above)
Comment (lluiscampos) [reply 04.System-updates-Debian-family/99.Variables/docs.md:132]: Maybe "add-on" is the correct term, actually. But I have already used addon in mender-convert code...

I'll change here but leave as-is in mender-convert

**Pull request merged**